CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # edge-react-gui
 
+## 2.19.0 (2022-08-01)
```

Comment:
We also added the `New Fiat Plugin framework` and fixed another crash when tapping on a wallet 'too' fast after login.